OwlcatTemplates.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <PackageType>Template</PackageType>
-    <PackageVersion>1.9.3</PackageVersion>
+    <PackageVersion>1.9.4</PackageVersion>

```

Comment:
As a staunch Defender of SemVer you're releasing two new templates as a patch version?